_posts/2016-03-16-code-smells-part-I.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+name: code-smells-part-one
+title: Code Smells – Part I
+date: 2016-03-16 12:10:00 +00:00
+author: Ana Nogal
+canonical:
+    name: my personal blog
+    href: http://www.ananogal.com/blog/code-smells-part-one/
+image:
+    src: /assets/img/custom/blog/code-smells.jpg
+    attribution:
+        text: Via Flickr/Creative Commons/Brian Fitzgerald
+tags:
+- bloaters
+- code smells
+- long classes/methods
+---
+Last weekend I was at the [SoCraTes Canaries](https://twitter.com/hashtag/socracan16) and I gave my first talk ever...oh boy! How nervous was I! But now that that has passed I was wondering what I should do with all information I collected: and then I thought, maybe it's a good idea to put it all in a nice blog post...well I guess I'll do it in more than one because it's a very big topic. Essentially if you want to develop in some subtopics as I do. So let's start and see where this can lead us to.
```

Comment:
- Last week I **attended** Socrates Canaries 
- ... **O**h boy!
- Essentially if you want to develop in some subtopics as I do (_ I don't understand this sorry_ )
- But now that that has passed I was wondering what I should do with all information I collected: and then I thought, maybe it's a good idea to put it all in a nice blog post...well I guess I'll do it in more than one because it's a very big topic. Essentially if you want to develop in some subtopics as I do. So let's start and see where this can lead us to. ( _ Blog smell? :-) Your opening paragraph should introduce your topic - not about whether you might or might not stick with one blog or many blogs_ )